classes/helper.php

```diff
@@ -46,6 +46,21 @@ public static function scripttypeasstring(int $calltype): string {
             case profile::SCRIPTTYPE_WS:
                 return get_string('excimertype_ws', 'tool_excimer');
         }
+        throw new \coding_exception('Unsupported script type ' . $type);
```

Comment:
if it is just display logic which isn't critical, I'd make this kind of code more forgiving and just print an 'unknown' instead of exploding

Reply:
Done